app/lib/services/ddpSocket.test.ts

```
@@ -0,0 +1,183 @@
+
+		const a = socket.forceReopen();
+		const b = socket.forceReopen();
+
+		expect(openMock).toHaveBeenCalledTimes(1);
+
+		resolveOpen();
+		await Promise.all([a, b]);
+		expect(openMock).toHaveBeenCalledTimes(1);
+	});
+
+	it("emits 'close' synchronously before opening so app-level Redux disconnect dispatches", async () => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)